Newsletters/_posts/Yearn-Finance-Newsletter-50/2021-12-05-Yearn-Finance-Newsletter-50.md

```diff
@@ -4,111 +4,111 @@ title:  "Yearn Finance Newsletter #50"
+Existem atualmente 5 opções para votação, com os nomes de trabalho sendo ySplit, veYFI, xYFI, BurnKeYs e Status Quo. A [proposta ySplit](https://docs.google.com/document/d/1dAWTkS_ZsXNy7mKKjOFUjILSlLsLz9KhGfLrwVu0GUg/edit) sugere a divisão do token YFI de 10.000 a 1.000.000 para combater o viés da unidade. A [proposta veYFI](https://docs.google.com/document/d/1hoi-IVccOB6iUJYzuApVbyjbQBx8-M0UuzZosb9wlWM/edit) sugere o bloqueio de YFI por um determinado período de tempo para receber direitos de governança que ditam para quais vaults os ganhos da tesouraria devem ir. [xYFI](https://docs.google.com/document/d/1ev16BXu3bDC8zMSBvHmxMWIeD82ptZck6SJAO5frV5g/edit) sugere a criação de um vault de staking de YFI para onde os lucros da tesouraria excedentes devem ser enviados. [BurnKeYs](https://docs.google.com/document/d/1BqmRsfdfCIaCtNZULdhKqUJzpKdaHE1XOGQlVp2nuSc/edit) exige que a função mint do token YFI seja permanentemente desativada, limitando assim o fornecimento em 36.666 tokens. O Status Quo deixaria o modelo econômico do token (tokenomics) YFI em seu estado atual e informa os titulares e potenciais investidores sobre os benefícios.
 
-Vote here at [the Snapshot page](https://yearn.snapshot.page/#/proposal/0x783cb3d57dd59b2827f6a42967375f06504cc947ebaa3c0e495c7b29ffd47aea). Thank you to all in the community who have contributed. An update from the working group, haiku included, can be found [here](https://docs.google.com/document/d/1-YEfXqXgTm-qzhPRUKs5allfX1XqYUOYwr_49FApnLU/edit).
+Vote aqui na [página Snapshot](https://yearn.snapshot.page/#/proposal/0x783cb3d57dd59b2827f6a42967375f06504cc947ebaa3c0e495c7b29ffd47aea). Obrigado a todos na comunidade que contribuíram. Uma atualização do grupo de trabalho, haiku incluído, pode ser encontrada [aqui](https://docs.google.com/document/d/1-YEfXqXgTm-qzhPRUKs5allfX1XqYUOYwr_49FApnLU/edit).
 
 ## Yearn Docs
 
-Several months ago Yearn contributors set out to rewrite our docs to make them friendly and approachable.
+Vários meses atrás, os contribuidores do Yearn decidiram reescrever nossos documentos para torná-los amigáveis e acessíveis.
 
-The completely refreshed docs are here to guide users, devs, partners, and to feed your curiosity with many previously unseen entries.
+Os documentos completamente atualizados estão aqui para orientar usuários, desenvolvedores, parceiros e para alimentar sua curiosidade com muitas entradas nunca antes vistas.
 
-Check out the new docs [here](https://docs.yearn.finance/).
+Confira os novos documentos [aqui](https://docs.yearn.finance/).
 
-## YIP-64 Fails To Meet Quorum
+## O YIP-64 não atinge o quorum
```

Comment:
is the 0 supposed to be there?

Reply:
Hi. This is the letter "O", not the number "0". It has the meaning of "The", like: "The YIP-64 Fails to Meet the Quorum". Is not a "must-have", but it's ok to be there.